hs_core/hydroshare/hs_bagit.py

```diff
@@ -21,20 +22,14 @@ def delete_bag(resource):
     # delete the resource bag
     istorage.delete('bags/{res_id}.zip'.format(res_id=res_id))
 
-def create_bag(resource):
-    """
-    Modified to implement the new bagit workflow. The previous workflow was to create a bag from the current filesystem
-    of the resource, then zip it up and add it to the resource. The new workflow is to delegate bagit and zip-up
-    operations to iRODS, specifically, by executing an iRODS bagit rule to create bagit file hierarchy, followed by
-    execution of an ibun command to zip up the bagit file hierarchy.
-
-    Note, this procedure may take awhile.  It is highly advised that it be deferred to a Celery task.
-
-    Parameters:
-    :param resource: (subclass of AbstractResource) A resource to create a bag for.
+    # delete the bags table
+    for bag in resource.bags.all():
+        try:
+            bag.delete()
+        except Exception as ex:
+            raise Exception(ex.message)
```

Comment:
@pkdash @ironfroggy The previous code used try/except for bag.delete, but rather than raise an exception, it just ignores/suppresses the error by using "pass" in except: block. I thought it is better to raise the error, but now that I think about it more, the error is raised anyway, so I should not add another layer. So I have removed try/except block here and pushed the code. If any of you see we should ignore the exception here as we did before, let me know as I think we should raise the exception here rather than ignore it.